static/templates/compose-invite-users.handlebars

```diff
@@ -1,5 +1,5 @@
 <div class="compose_invite_user" data-useremail="{{email}}">
-    {{#tr this}}<strong>__name__</strong> is not subscribed to this stream.{{/tr}}
+    {{#tr this}}<strong>__name__</strong> is not subscribed to this stream. This user won't be notified if you mention them here.{{/tr}}
```

Comment:
I think "They will not be notified ..." is better.

Can you also squash the two commits, and fix the commit message to match our style?

Check out the Zulip commit message guidelines for more details: https://zulip.readthedocs.io/en/latest/contributing/version-control.html#commit-messages